### DISCUSSION TITLE
New Database and others games save location

### DISCUSSION THREAD

**Discussion by Alkhymia** [mtkennerly/ludusavi#239] [Q&A] [answered]
Hi thanks for this amazing project!

Some questions i have:

Can you add https://steamdb.info/?

And dont know why i have games saved on:

C:\Users\Public\Documents\Steam\CODEX\1718570 (for example this game isnt in pcgamingwiki)

And when I preview I didn't find anything but if I add custom they show up.

Regards

**Comment by mtkennerly** [comment, marked as answer]
Hi!

1. The [Ludusavi manifest](https://github.com/mtkennerly/ludusavi-manifest) already directly queries Steam for some information, so I could potentially scrape the save paths like SteamDB does. There are some limitations with this, but I think it's worth exploring. I've opened a ticket that you can follow for any progress updates: https://github.com/mtkennerly/ludusavi-manifest/issues/29

2. From Google, it looks like CODEX is a group that releases cracked games. Ludusavi doesn't support cracks (I'd rather not open myself up to any legal liability), but yes, you can add custom game entries for anything that you want.
